.github/workflows/check.yml

```diff
@@ -137,7 +151,7 @@ jobs:
             *) continue ;; # skip others
             esac
 
-            cppcheck --force -j4 --suppressions-list=.github/workflows/cppcheck-spr-list --quiet --enable=warning -iheaplayers $f 2>>$result_file
+            ${{ env.CPPCHECK_PATH }}/cppcheck --force -j4 --suppressions-list=.github/workflows/cppcheck-spr-list --quiet --enable=warning -iheaplayers $f 2>>$result_file
```

Comment:
I've run across some errors when "make install" cppcehck, so I use the local executable file which is built.

Reply:
If using the local executable, please note that cppcheck also needs the `cfg` directory from the cppcheck source tree. If the directory is copied near the newly compiled executable, it works.

Reply:
Thank you for your advice. When I installed cppcheck by using "make install", the error you mentioned occurs, which says  "std.cfg not found". So, I just tried to use the local executable like this, and it seems to work. 

> If the directory is copied near the newly compiled executable
I don't quite follow you. Could you elaborate on it?

Reply:
in the cppcheck source tree, there is a `cfg` directory with - I presume - default settings.
I found on SO a mention that the simplest to make it work (without a `make install`) is to just copy the `cfg` directory near the newly build binary:
```
~/tmp/cppcheck_build/bin$ tree
.
├── cfg
│   ├── avr.cfg
│   │   ...
│   ├── std.cfg
│   │   ...
│   └── zlib.cfg
└── cppcheck
```
This might not be relevant if the script does a `make install`.

Reply:
After building cppcheck in this script, the executable is created in the cppcheck source directory which includes the `cfg`. The location of the cppcheck source directory is assigned to `${{ env.CPPCHECK_PATH }}`. So, there seems to be no problem. What do you think?

Reply:
Well, I understand now.
So, it worked because the build was done in-tree.
In that case, please disregard my comment.

Reply:
OK. Thank you for the review :).